src/component/NavBar.css

```diff
@@ -0,0 +1,22 @@
+.logo {
+  width: 288px;
+  height: 88px;
+  flex-shrink: 0;
+  background: lightgray 50% / cover no-repeat;
+}
+
+.navbar {
+  width: 1280px;
+  height: 97px;
+  flex-shrink: 0;
+  background: #FFF;
+  box-shadow: 0px 4px 12px 0px rgba(0, 0, 0, 0.25);
+}
+
+.navBtn {
```

Comment:
the styling should match the figma design. you can use the text-decoration property to remove the blue color of anchor tag.